src/main/java/core/presenters/levels/LevelGameplayController.java

```diff
@@ -38,6 +38,8 @@
     private Box2DDebugRenderer box2DDebugRenderer;
     private HudManager hud;
     private InputController inputController;
+    static final int SWORD_LEVEL = 1;
+    static final int DAGGER_LEVEL = 1;
```

Comment:
these are for demo purposes, in fact these things should happen by adding a callback to the Player spawner in `DemoSpawners.createPlayerSpawner`

Reply:
Not sure if I understand which exact parts of the code need to be refactored

Reply:
should happen in a player spawn callback